### PR TITLE
Dockeree

### PR DESCRIPTION
**Pull request by Arau**
First draft to review structure, message, etc.

### CODE REVIEW

File: _docs/platforms/dockeree/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+# DockerEE
+
+This section of documentation covers the use of StorageOS in Docker Enterprise Edition.
+
+DockerEE and the Universal Control Plane can be executed in different Linux
+distributions. StorageOS supports RHEL, CentOS, Debian, and selected Ubuntu
+images. For more details, check out the supported OS in the [prerequisites page]({%link _docs/prerequisites/systemconfiguration.md %}).
+
+StorageOS supports only Kubernetes nodes managed by Docker Enterprise Edition.
```
Comment (avestuk) [inline _docs/platforms/dockeree/index.md]:
```suggestion
StorageOS only supports Kubernetes nodes managed by Docker Enterprise Edition.
```

File: _includes/k8s/bestpractices/swarm-nodes.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## StorageOS in DockerEE
+
+It is recommended to use Kubernetes nodes to provision Stateful workloads as
+StorageOS is not supported to run in Swarm nodes. StorageOS volumes have to be
```
Comment (avestuk) [inline _includes/k8s/bestpractices/swarm-nodes.md]:
```suggestion
as the running of StorageOS is not supported in Swarm nodes. StorageOS volumes have to be
```

File: _docs/platforms/dockeree/index.md (hunk 1 above)
Comment (Arau) [reply _docs/platforms/dockeree/index.md]: Thank you.

Done

File: _docs/platforms/dockeree/firstvolume/index.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+layout: guide
+title: StorageOS Docs - dockeree
+anchor: platforms
+cmd: kubectl
+platform: dockeree
+platformUC: DockerEE
```
Comment (actionbuddha) [inline _docs/platforms/dockeree/firstvolume/index.md]: Docker EE (add space) ?

File: _docs/platforms/dockeree/index.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+layout: guide
+title: StorageOS Docs - DockerEE
+anchor: platforms
+module: platforms/dockeree
+---
+
+# DockerEE
```
Comment (actionbuddha) [inline _docs/platforms/dockeree/index.md]: Docker EE

Comment (actionbuddha) [reply _docs/platforms/dockeree/index.md]: (and other places)

File: _docs/platforms/dockeree/index.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+This section of documentation covers the use of StorageOS in Docker Enterprise Edition.
+
+DockerEE and the Universal Control Plane can be executed in different Linux
```
Comment (actionbuddha) [inline _docs/platforms/dockeree/index.md]: s/can be executed in/can run on/

File: _docs/platforms/dockeree/index.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+distributions. StorageOS supports RHEL, CentOS, Debian, and selected Ubuntu
+images. For more details, check out the supported OS in the [prerequisites page]({%link _docs/prerequisites/systemconfiguration.md %}).
+
+StorageOS only supports Kubernetes nodes managed by Docker Enterprise Edition.
+Therefore, StorageOS volumes can be provisioned and used only by applications
+on those nodes.
+
+> Mixed nodes support is experimental
```
Comment (actionbuddha) [inline _docs/platforms/dockeree/index.md]: I think we should be stronger here. Nodes running a mix of Kubernetes/Docker Swarm workloads are unsupported? I really don't want to get bogged down in endless support calls by people attempting to mount volumes from within Swarm.

File: _includes/k8s/bestpractices/swarm-nodes.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+## StorageOS in DockerEE
+
+It is recommended to use Kubernetes nodes to provision Stateful workloads as
```
Comment (actionbuddha) [inline _includes/k8s/bestpractices/swarm-nodes.md]: I think we should move this comment to the root of the Docker EE section, or duplicate it there. Let's not bury it in a Best Practice but shout it clearly.

**Comment by actionbuddha** [comment]
One other comment, can we rename the community edition of Docker to 'Docker CE' to clearly differentiate them?

File: _docs/platforms/dockeree/firstvolume/index.md (hunk 3 above)
Comment (Arau) [reply _docs/platforms/dockeree/firstvolume/index.md]: Done

File: _docs/platforms/dockeree/index.md (hunk 4 above)
Comment (Arau) [reply _docs/platforms/dockeree/index.md]: done

File: _docs/platforms/dockeree/index.md (hunk 5 above)
Comment (Arau) [reply _docs/platforms/dockeree/index.md]: done

File: _docs/platforms/dockeree/index.md (hunk 6 above)
Comment (Arau) [reply _docs/platforms/dockeree/index.md]: Done

File: _includes/k8s/bestpractices/swarm-nodes.md (hunk 7 above)
Comment (Arau) [reply _includes/k8s/bestpractices/swarm-nodes.md]: Agreed

**Comment by Arau** [comment]
> One other comment, can we rename the community edition of Docker to 'Docker CE' to clearly differentiate them?

Done and all the links, etc repointed

**Comment by avestuk** [comment]
LGTM, checked links and looks good.

File: _docs/platforms/dockeree/install/index.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+Docker EE and the Universal Control Plane can be executed in different Linux
+distributions. StorageOS supports RHEL, CentOS, Debian, and selected Ubuntu
+images. For more details, check out the supported OS in the [prerequisites page]({%link _docs/prerequisites/systemconfiguration.md %}).
+
+StorageOS only supports Kubernetes nodes managed by Docker Enterprise Edition.
+Therefore, StorageOS volumes can be provisioned and used only by applications
+on those nodes. Therefore, its recommended to use Kubernetes nodes to provision
+Stateful workloads.
+
+> Mixed nodes (Kuberntes + Swarm) are not supported to run StorageOS.
```
[the remark this answers is not in the document]
Comment (actionbuddha) [reply _docs/platforms/dockeree/install/index.md]: I'm hoping my last pushed commits (4202c90) reword this slightly.

[the remark this answers is not in the document]
Comment (Arau) [reply _docs/platforms/dockeree/install/index.md]: Last change looks good to me.

Thank you everyone!

**Pull request merged**